src/Nest/Ingest/Processor/ElasticClient-IngestProcessorGrok.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Elasticsearch.Net;
+
+namespace Nest
+{
+	using System.Threading;
+	using NodesHotThreadConverter = Func<IApiCallDetails, Stream, NodesHotThreadsResponse>;
```

Comment:
`using` directives should be outside the namesace in the file. Also, doesn't look like the `using` alias is needed here?

Reply:
Copy pasta. Fixed.